manifests/server.pp

```diff
@@ -18,6 +18,14 @@
 #
 # $autosign_mode::             mode of the autosign file/script
 #
+# $autosign_content::          If set, write the autosign file content
+#                              using the value of this parameter. 
+#                              Cannot be used at the same time as autosign_entries
+#                              For example, could be a string, or
+#                              file('another_module/autosign.sh') or
+#                              template('another_module/autosign.sh.erb')
+#                              type:string
```

Comment:
hm, we don't have new style typehints at that subclass anyway, so leave it as is, sorry